### PR TITLE
Bug26367 035 01

### PR DESCRIPTION
**Pull request by dgoulet-tor**

### CODE REVIEW

File: src/core/or/or.h [hunk 1]
```diff
@@ -103,10 +103,6 @@ struct curve25519_public_key_t;
 #define cell_t tor_cell_t
 #endif
 
-#ifdef ENABLE_TOR2WEB_MODE
-#define NON_ANONYMOUS_MODE_ENABLED 1
```
Comment (teor2345) [inline src/core/or/or.h]: Please delete all references to NON_ANONYMOUS_MODE_ENABLED as well.

File: src/feature/rend/rendclient.c [hunk 2]
```diff
@@ -1226,15 +1221,12 @@ rend_parse_service_authorization(const or_options_t *options,
 
 /* Can Tor client code make direct (non-anonymous) connections to introduction
  * or rendezvous points?
- * Returns true if tor was compiled with NON_ANONYMOUS_MODE_ENABLED, and is
- * configured in Tor2web mode. */
+ * Returns true if tor was compiled with NON_ANONYMOUS_MODE_ENABLED. */
```
Comment (teor2345) [inline src/feature/rend/rendclient.c]: Please delete all references to NON_ANONYMOUS_MODE_ENABLED as well.

File: src/feature/rend/rendclient.c [hunk 3]
```diff
@@ -1226,15 +1221,12 @@ rend_parse_service_authorization(const or_options_t *options,
...
 int
 rend_client_allow_non_anonymous_connection(const or_options_t *options)
 {
-  /* Tor2web support needs to be compiled in to a tor binary. */
 #ifdef NON_ANONYMOUS_MODE_ENABLED
```
Comment (teor2345) [inline src/feature/rend/rendclient.c]: Please delete all references to NON_ANONYMOUS_MODE_ENABLED as well.

File: src/feature/rend/rendclient.c [hunk 4]
```diff
@@ -1247,9 +1239,7 @@ int
 rend_client_non_anonymous_mode_enabled(const or_options_t *options)
 {
   (void)options;
-  /* Tor2web support needs to be compiled in to a tor binary. */
 #ifdef NON_ANONYMOUS_MODE_ENABLED
```
Comment (teor2345) [inline src/feature/rend/rendclient.c]: Please delete all references to NON_ANONYMOUS_MODE_ENABLED as well.

File: src/feature/rend/rendclient.c [hunk 5]
```diff
@@ -1247,9 +1239,7 @@ int
 rend_client_non_anonymous_mode_enabled(const or_options_t *options)
```
Comment (teor2345) [inline src/feature/rend/rendclient.c]: Please delete all references to NON_ANONYMOUS_MODE_ENABLED as well.

**Comment by nmathewson** [comment]
This branch was squashed and merged

**Pull request closed without merging**